patient-appointment-management/.env

```diff
@@ -0,0 +1,88 @@
+# description: Customer name to be displayed in SMS.
+# format: text
+# required: true
+# configurable: true
+CUSTOMER_NAME=
+
+# description: Customer short name. Must be all lowercase one word without any spaces.
+# format: text
+# required: true
+# configurable: true
+APPLICATION_CUSTOMER_CODE=
+
+# description: Customer EHR endpoint url for communication from Twilio via HTTP POST
```

Comment:
Is there a link available that explains what an EHR is? Usually we'd include external documentation like that as a `# link: <url>` argument to the variable definition.

Reply:
EHR standards for Electronic Health Records and is ubiquitously used acronym in Healthcare. We may want to expand out the acronym in longer descriptions of the application in case, some non-healthcare customer looks at it.

Reply:
I will let our product manager (Katie) know. She is creating all the external facing documentation.

Reply:
Hey all - I will make sure to be clear in both the blog post, QD app landing page and all other customer-facing documentation who this app is intended for (i.e. explicitly for healthcare organizations) and that Electronic Health Record (EHR) integration is required in order for the app to work. I agree with BJ here though that the concept of EHR is ubiquitous in Healthcare and whoever is implementing this and integrating with their EHR will know what the EHR endpoint URL is. Happy to expand upon this in some more detail in the Integration Guide to be sure!